src/test/java/com/neo4j/docker/TestConfSettings.java

```diff
@@ -16,123 +16,187 @@
-
-    private GenericContainer createContainer()
-    {
-        return new GenericContainer( TestSettings.IMAGE_ID )
-                .withEnv( "NEO4J_AUTH", "none" )
-                .withEnv( "NEO4J_ACCEPT_LICENSE_AGREEMENT", "yes" )
-                .withExposedPorts( 7474, 7687 )
-                .withLogConsumer( new Slf4jLogConsumer( log ) );
+public class TestConfSettings {
+    private static Logger log = LoggerFactory.getLogger(TestConfSettings.class);
+
+    private GenericContainer createContainer() {
+        return new GenericContainer(TestSettings.IMAGE_ID)
+                .withEnv("NEO4J_AUTH", "none")
+                .withEnv("NEO4J_ACCEPT_LICENSE_AGREEMENT", "yes")
+                .withExposedPorts(7474, 7687)
+                .withLogConsumer(new Slf4jLogConsumer(log));
     }
 
     @Test
-    void testIgnoreNumericVars()
-    {
+    void testIgnoreNumericVars() {
         GenericContainer container = createContainer();
-        container.withEnv( "NEO4J_1a", "1" );
+        container.withEnv("NEO4J_1a", "1");
         container.start();
-        Assertions.assertTrue( container.isRunning() );
+        Assertions.assertTrue(container.isRunning());
 
         WaitingConsumer waitingConsumer = new WaitingConsumer();
         container.followOutput(waitingConsumer);
 
-        Assertions.assertDoesNotThrow( () -> waitingConsumer.waitUntil( frame -> frame.getUtf8String()
-                                               .contains( "WARNING: 1a not written to conf file because settings that " +
-                                                          "start with a number are not permitted" ), 15, TimeUnit.SECONDS ),
-                                       "Neo4j did not warn about invalid numeric config variable `Neo4j_1a`" );
+        Assertions.assertDoesNotThrow(() -> waitingConsumer.waitUntil(frame -> frame.getUtf8String()
+                        .contains("WARNING: 1a not written to conf file because settings that " +
+                                "start with a number are not permitted"), 15, TimeUnit.SECONDS),
+                "Neo4j did not warn about invalid numeric config variable `Neo4j_1a`");
         container.stop();
     }
 
-    private Map<String, String> parseConfFile( File conf) throws FileNotFoundException
-    {
-        Map<String,String> configurations = new HashMap<>(  );
-        Scanner scanner = new Scanner( conf );
-        while (scanner.hasNextLine())
-        {
-            String[] params = scanner.nextLine().split( "=", 2 );
-            log.debug( params[0] +"\t:\t"+ params[1] );
-            configurations.put( params[0], params[1] );
+    private Map<String, String> parseConfFile(File conf) throws FileNotFoundException {
+        Map<String, String> configurations = new HashMap<>();
+        Scanner scanner = new Scanner(conf);
+        while (scanner.hasNextLine()) {
+            String[] params = scanner.nextLine().split("=", 2);
+            log.debug(params[0] + "\t:\t" + params[1]);
```

Comment:
`testAdminConfOverride()` failed here when I ran it.
Change this while loop to:
```
while ( scanner.hasNextLine() )
        {
            String[] params = scanner.nextLine().split( "=", 2 );
            if(params.length < 2)
            {
                continue;
            }
            log.debug( params[0] + "\t:\t" + params[1] );
            configurations.put( params[0], params[1] );
        }
```